src/ert/job_queue/queue.py

```diff
@@ -113,11 +111,10 @@ def __init__(self, driver: "Driver", max_submit: int = 2):
 
         self.job_list: List[JobQueueNode] = []
         self._stopped = False
-        c_ptr = self._alloc(STATUS_file, ERROR_file)
```

Comment:
These seem to be unused, were these files supposed to be used? Residue or missing feature?